perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -375,7 +372,7 @@ def _mount_gcs_bucket(bucket_name) -> str:
   subprocess.call('mkdir {}'.format(gcs_bucket), shell=True)
 
   exit_code = subprocess.call(
-      'gcsfuse --implicit-dirs --disable-http2 --max-conns-per-host 100 {} {}'.format(
+      'go run . --implicit-dirs --disable-http2 --max-conns-per-host 100 {} {}'.format(
```

Comment:
You can install gcsfuse in build.sh in the similar way as done here: https://github.com/GoogleCloudPlatform/gcsfuse/blob/461333abf506eb6a0926c84b3f4c3c59323b44a5/tools/package_gcsfuse_docker/Dockerfile#L38. 
You can keep the commit ID as GCSFuse version while installing GCSFuse.

That way, you will not need to do this and L478 changes.

Reply:
fixed